src/Html/Column.php

```diff
@@ -34,7 +34,7 @@ public function __construct($attributes = [])
         // Allow methods override attribute value
         foreach ($attributes as $attribute => $value) {
             $method = 'parse' . ucfirst(strtolower($attribute));
-            if (method_exists($this, $method)) {
+            if ($value && method_exists($this, $method)) {
```

Comment:
Not sure on this one. I think it might break if the value is zero `0`? Maybe `!is_null($value)` might work?

Reply:
That's right! I've updated the PR.